test/suites/standard1/configuration.robot

```diff
@@ -110,3 +155,23 @@ Setup Audit Flags
     ${merged}=    Extend MicroShift Config    ${AUDIT_FLAGS}
     Upload MicroShift Config    ${merged}
     Restart MicroShift
+
+Deploy Storage Config
+    [Documentation]    Applies a storage ${config} to the exist MicroShift config, pushes it to the MicroShift host,
+    ...    and restarts microshift.service
+    [Arguments]    ${config}
+    Cleanup MicroShift    opt='--keep-images'
```

Comment:
Is this required because `${config}` may disable csi components and you need to delete those pods (hence removing etcd)? This relates to the comment above in the code.

Reply:
`--keep-images` just makes the cleanup script invoke `crictl rmi --all`, which deletes the images, not containers. This was just to lessen the runtime of the tests between wipes and restarts.